.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: r
+cache: packages
+sudo: false
+r_packages:
+  - readxl
+  - rjson
+  - stringr
+  - org.Hs.eg.db
+  - xlsx
+  - tm
+  - slam
+before_install:
+  - cd code
```

Comment:
this should probably be under `script:` section

Reply:
OK, between before_install, after_success, script... not always clear what goes where...